docs/source/algorithms/SortTableWorkspace-v1.rst

```diff
@@ -42,15 +42,15 @@ Usage
 
     # Sort in ascending order
     sorted_asc = SortTableWorkspace(table, Columns = ['x','y','z'])
-    print 'Sorted ascending'
+    print('Sorted ascending')
     for i in range(sorted_asc.rowCount()):
-        print sorted_asc.row(i)
+        print(sorted_asc.row(i))
```

Comment:
These tests are printing a dict which you then can't assume the order of the columns for python3
```
Expected:
    Sorted ascending
    {'y': 'one (1)', 'x': 1, 'z': 7.0}
    {'y': 'one (2)', 'x': 1, 'z': 2.0}
    {'y': 'one (3)', 'x': 1, 'z': 1.0}
    {'y': 'two (1)', 'x': 2, 'z': 3.0}
    {'y': 'two (1)', 'x': 2, 'z': 8.0}
    {'y': 'two (2)', 'x': 2, 'z': 6.0}
    {'y': 'two (2)', 'x': 2, 'z': 9.0}
    {'y': 'three (2)', 'x': 3, 'z': 4.0}
    {'y': 'three (2)', 'x': 3, 'z': 5.0}
    {'y': 'three (3)', 'x': 3, 'z': 0.0}
    Sorted descending
    {'y': 'three (3)', 'x': 3, 'z': 0.0}
    {'y': 'three (2)', 'x': 3, 'z': 5.0}
    {'y': 'three (2)', 'x': 3, 'z': 4.0}
    {'y': 'two (2)', 'x': 2, 'z': 9.0}
    {'y': 'two (2)', 'x': 2, 'z': 6.0}
    {'y': 'two (1)', 'x': 2, 'z': 8.0}
    {'y': 'two (1)', 'x': 2, 'z': 3.0}
    {'y': 'one (3)', 'x': 1, 'z': 1.0}
    {'y': 'one (2)', 'x': 1, 'z': 2.0}
    {'y': 'one (1)', 'x': 1, 'z': 7.0}
Got:
    Sorted ascending
    {'x': 1, 'y': 'one (1)', 'z': 7.0}
    {'x': 1, 'y': 'one (2)', 'z': 2.0}
    {'x': 1, 'y': 'one (3)', 'z': 1.0}
    {'x': 2, 'y': 'two (1)', 'z': 3.0}
    {'x': 2, 'y': 'two (1)', 'z': 8.0}
    {'x': 2, 'y': 'two (2)', 'z': 6.0}
    {'x': 2, 'y': 'two (2)', 'z': 9.0}
    {'x': 3, 'y': 'three (2)', 'z': 4.0}
    {'x': 3, 'y': 'three (2)', 'z': 5.0}
    {'x': 3, 'y': 'three (3)', 'z': 0.0}
    Sorted descending
    {'x': 3, 'y': 'three (3)', 'z': 0.0}
    {'x': 3, 'y': 'three (2)', 'z': 5.0}
    {'x': 3, 'y': 'three (2)', 'z': 4.0}
    {'x': 2, 'y': 'two (2)', 'z': 9.0}
    {'x': 2, 'y': 'two (2)', 'z': 6.0}
    {'x': 2, 'y': 'two (1)', 'z': 8.0}
    {'x': 2, 'y': 'two (1)', 'z': 3.0}
    {'x': 1, 'y': 'one (3)', 'z': 1.0}
    {'x': 1, 'y': 'one (2)', 'z': 2.0}
    {'x': 1, 'y': 'one (1)', 'z': 7.0}
```